docs/pbc/all/dynamic-multistore/202410.0/base-shop/difference-between-modes.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Difference in deployment files
+description: This document describes the difference between Dynamic Multistore ON and OFF mode.
```

Comment:
Maybe move this too docs/pbc/all/dynamic-multistore/202410.0/base-shop/difference-between-modes.md

Reply:
It is the same page

Reply:
Sorry, wrong reference